app/views/posts/index.html.erb

```diff
@@ -1,10 +1,13 @@
 <%= render 'shared/header' %>
 <%= link_to 'New post', '/posts/new' %>
 
+<p>Your location: <%= @location.first %>, <%= @location.last %></p>
```

Comment:
kinda strange but okay, maybe wanna remove or change to current_user distance to this post or something

Reply:
temporary and agreed, will come in later pr